src/test/java/picard/sam/markduplicates/UmiAwareMarkDuplicatesWithMateCigarTest.java

```diff
@@ -223,23 +226,25 @@ public void testBadUmis(List<String> umis, List<String> assignedUmi, final List<
                 Arrays.asList(new String[]{"AAAA", "AAAA", "ATTA", "AAAA", "AAAA"}), // Expected inferred UMI
```

Comment:
The explicit array creations in this file are unnecessary, e.g. you can delete `new String[]` from this expression.